resources/less/app.less

```diff
@@ -0,0 +1,10 @@
+
+@import "~bootstrap/less/bootstrap";
+@import "variables";
+@import "bootswatch";
+@import '~datatables.net-bs/css/dataTables.bootstrap.css';
+@import '~datatables.net-responsive-bs/css/responsive.bootstrap.css';
+
+body, label, .checkbox label {
+	font-weight: 300;
+}
```

Comment:
Ik had deze weggehaald in mijn `feature/skills` branch omdat alle form labels in heel AAS hierdoor niet meer dikgedrukt zijn. Als je dit nodig hebt voor je declaratie componenten dan moet je de class specifieker maken.

Reply:
Dit heb ik niet nodig, het is een artifact van oude less files